### ISSUE TITLE
Vue component appears to be parsing mustache tags within code snippets

### ISSUE THREAD

**Issue by henryjin3** [hmsk/frontmatter-markdown-loader#154]
When using frontmatter-markdown-loader in Vue component mode, code examples from the markdown file that are in handlebars seem to be getting parsed out. For example:

```html
<h1>{{ title }}</h1>
```

The header will show up blank. And in another case:

```html
<template>
  <div>
    writing index
    <section>
      <ol>
        <li v-for="post in posts" :key="post.attributes.title">
          <nuxt-link :to="post.path">
            {{ post.attributes.title }}
          </nuxt-link>
        </li>
      </ol>
    </section>
  </div>
</template>
```
The component is failing to render, saying that "attributes is undefined."

You can see my current setup at https://github.com/henryjin3/henryjin-dev/tree/vuemarkdown. 

Edit: forgot to mention, this only happens when I switch to Vue component mode, and using standard HTML mode with markdown-it works just fine.

**Comment by hmsk** [comment]
Thanks for reporting :)
This seems a bug, I will look into it. Btw, your blog system is excellent ✨

**Comment by hmsk** [comment]
Released the fix for such the problem as `v3.4.3-0`, could you play with this version? @henryjin3

**Comment by henryjin3** [comment]
Looks good to me, thanks so much! I even debugged through the regex code but didn't realize that was the source of the issue :frowning:.

**Comment by hmsk** [comment]
Your blog article data helped me to find the exact issue. Prism tweaks `<code>` contents with inline HTML and I didn't predict such the case when I implemented originally. Anyways, the original regex was really fragile though 🙈 (but don't want to have HTML parser just for Vue feature at this time)

Thanks for providing reproducible codes 💖 
I'm going to release as `v3.4.3` after having proper tests for this problem.

**Comment by hmsk** [comment]
Released as `v3.4.3` ✨
